tutorials/60-minute-blitz.jl

```diff
@@ -119,17 +119,17 @@ f(5)
 # In simple cases it's pretty easy to work out the gradient by hand – here it's
 # `6x+2`. But it's much easier to make Flux do the work for us!
 
-using Flux.Tracker: derivative
+using Flux.Tracker: gradient
 
-df(x) = derivative(f, x)
+df(x) = gradient(f, x)
 
 df(5)
 
 # You can try this with a few different inputs to make sure it's really the same
 # as `6x+2`. We can even do this multiple times (but the second derivative is a
 # fairly boring `6`).
 
-ddf(x) = derivative(df, x)
+ddf(x) = gradient(df, x)
 
 ddf(5)
```

Comment:
This does not work on Julia 1.10 and Flux 0.8.3 
See also 
https://github.com/FluxML/Zygote.jl/issues/278